### ISSUE TITLE
RFE: add letters as accelerator for persistent entries

### ISSUE THREAD

**Issue by mat-m** [rickyrockrat/parcellite#17]
Actually the persistent stack show all lines like

> https://fsf.org
> This is copied from text
> parcellite

But having:

A. https://fsf.org
B. This is copied from text
C. parcellite

Where A,B & C would be underlined and reachable through key press, like a menu accelerator.
So, without mouse, I can do `Ctrl-Alt-H, A` to paste the FSF url.

A similar addition to add 1..9 0 to first ten history items would be great also.

**Comment by rickyrockrat** [comment]
I agree with the persistent stack, and I like the idea a lot. Getting it coded is a different story time wise.

**Comment by mat-m** [comment]
Thanks to acknowledge my request. I have no coding skill to help, sorry.
